source/content/guides/composer-convert.md

```diff
@@ -25,34 +36,69 @@ export site=yoursitename
 
 </Alert>
 
-## Checkout a New Branch
+## Add the Pantheon Integrated Composer Upstream in a New Local Branch
 
-You're about to make some massive changes to the codebase. We recommend you to do this work on a new branch, as it might take you some time to complete and rolling back can be complicated:
+You're about to make massive changes to the codebase. We recommend you to do this work on a new branch, as it might take you some time to complete and rolling back changes can be complicated:
 
 1. In your local terminal, change directories to your site project. For example, if you keep your projects in a folder called `projects` in the home directory:
 
   ```bash{promptUser:user}
   cd ~/projects/$site/
   ```
 
-2. Create the new branch:
+1. Add the Pantheon Drupal Upstream as a new remote, fetch the `ic` branch, and checkout to a new local branch based on it called `composerify`:
```

Comment:
`ic` is a remote, not a branch. The commands below add a new remote called ic, fetches its master branch and then creates a new branch called "composerify" from the master branch of the ic remote.

Note that the resulting branch "composerify" shares a commit history with the drupal-project upstream, and has no commits in common with the site's existing master branch.